Add support for VRC 720 & add "Live Monitor" statistics

AleXburnA opened the pull request:
I have added support for SensoComfort VRC 720 by simply symlinking the file for the 700.
(See https://github.com/john30/ebusd-configuration/issues/166)


I have also added support for some "Live Monitor" statistics for AroTherm VWL heatpumps.
(See https://github.com/john30/ebusd-configuration/issues/167)

Please bear with me in case anything is wrong with this pull request. 
I am not a programmer and I did this for the first time :)

Best regards
Alex

john30 commented on ebusd-2.1.x/de/vaillant/08.hmu.csv:
```diff
@@ -8,6 +8,12 @@ r,,EnergyCool,,,,,01,,,energy,,,,,,,,,,,,,,,
 r,,HoursCool,,,,,43,,,energy,,,,,,,,,,,,,,,
+
+# some more statistics (Live Monitor)
+r,,CurrentYieldPower,,,,,23,,,D1B,10,,,,,,,,,,,,,,,,,,,,
```
* please add a comment for this (what does it reveal?)
* please add a unit (e.g. kW)
this applies to all added lines

AleXburnA replied:
I made new commits and fixed all your requested changes. I have also added a new Value: "Water Throughput". As this is only my second time of using Git / GitHub, I am unsure what exactly I have to do to get this PR right. I have committed all my changes to this branch, can you see the changes?

john30 commented on ebusd-2.1.x/en/vaillant/08.hmu.csv:
```diff
@@ -8,6 +8,12 @@ r,,EnergyCool,,,,,01,,,energy,,,,,,,,,,,,,,,
 r,,HoursCool,,,,,43,,,energy,,,,,,,,,,,,,,,
+
+# some more statistics (Live Monitor)
+r,,CurrentYieldPower,,,,,23,,,D1B,10,,,,,,,,,,,,,,,,,,,,
```
see above

AleXburnA replied:
see above

AleXburnA commented:
sorry for the mess, I am in the learning phase of git/github and I am currently unsure how to clean this up.

This commit includes the following (all in 08.hmu.csv)
- Fixed CSV format for better editability
- Added units and better comments in DE and EN
- Added a new value: "Water Throughput"
